tests/Tree/Models/User.php

```diff
@@ -214,4 +215,31 @@ public function videosAndSelf(): MorphToManyOfDescendants
     {
         return $this->morphedByManyOfDescendantsAndSelf(Video::class, 'authorable');
     }
+
+    /**
+     * @return Attribute<string,never>
+     */
+    protected function displayPath(): Attribute
+    {
+        return Attribute::get(
+            fn (): string => (string) $this->ancestorsAndSelf
+                ->reverse()
```

Comment:
This one reverses the collection but is not called reverse{...} as a method name, the method below doesn't reverse the collection but is called reverse{...}, is this on purpose?